sensor_core/dsp/dsp_manager.py

```diff
@@ -1,3 +1,15 @@
 import numpy as np
+import scipy.signal as signal
 
-#TODO: add modules for 1D dsp 
\ No newline at end of file
+#Moving average filter
+def moving_avg_filter(data, N):
+    data_arr = np.concatenate(([np.min(data)] * (N), data))
+    cumsum = np.cumsum(data_arr)
+    yi = (cumsum[N:] - cumsum[:-N]) / float(N)
+    return yi
+    
+#Butterworth filter
+def butter_bandpass(data, min_frq, max_frq, N):
```

Comment:
Ideally we don't want to write multiple functions to use a butterworth filter for bandpass, LP, and HP.

Better to write it as a general buttworth filter with a parameter input for LPF, HPF, or BPF.

Reply:
Also might consider adding use case for notch filter